src/Components/test/testassets/BasicTestApp/FormsTest/InputFileComponent.razor

```diff
@@ -40,6 +40,8 @@ Max allowed files:
 <InputFile OnChange="LoadImage" id="input-image" />
 <br />
 
+<span id="exception-message">@exceptionMessage</span>
```

Comment:
Changes in this file aren't really related to the byte array interop changes, they're more so just to provide better visibility for exceptions, to make debugging these tests easier in the future. These changes ensure `LoadImage` follows the same pattern as `LoadFiles`.